flexget/plugins/sites/lostfilm.py

```diff
@@ -47,147 +37,128 @@ class LostFilm(object):
 
       lostfilm:
         url: <url>
+        pages: <number>
     """
 
     schema = {
         'type': ['boolean', 'object'],
         'properties': {
             'url': {'type': 'string', 'format': 'url'},
+            'pages': {'type': 'integer'}
         },
         'additionalProperties': False
     }
 
     def build_config(self, config):
-        """Set default url to config"""
+        """Set default url and pages config"""
         if isinstance(config, bool) and config is True:
-            config = {'url': LOSTFILM_URL}
+            config = {'url': NEW_RELEASES_URL, 'pages': 3}
         return config
 
     def on_task_input(self, task, config):
         config = self.build_config(config)
         if config is False:
```

Comment:
I'd rather that this was changed to
```
if not config:
    return
config = self.prepare_config(config)
```
I know this wasn't something you made, but you could just as well change it while you're in there. Also, historically, `build_config` is called `prepare_config`.

Reply:
@duhast You forgot this one